src/common/util.ts

```diff
@@ -32,12 +32,15 @@ export function executeFile(
         notUsePip = true
         return arg.replace(/%text/g, input.toString())
       }
+      if (/%filepath/.test(arg)) {
```

Comment:
I can't see why `%filename` is basenamed. This confuses things like .prettierignore. But in the name of backwards compatibility, I've added a separate placeholder

Reply:
Please also update the README of `Args additional syntax`